src/tcpip/tcpip.c

```diff
@@ -856,8 +857,9 @@ void mg_tcpip_init(struct mg_mgr *mgr, struct mg_tcpip_if *ifp) {
     mgr->extraconnsize = sizeof(struct connstate);
     if (ifp->ip == 0) ifp->enable_dhcp_client = true;
     memset(ifp->gwmac, 255, sizeof(ifp->gwmac));  // Set to broadcast
-    mg_random(&ifp->eport, sizeof(ifp->eport));   // Randomise the initial
-    ifp->eport += MG_EPHEMERAL_PORT_BASE;         // ephemeral port
+    mg_random(&ifp->eport, sizeof(ifp->eport));   // Random from 0 to 65536
+    ifp->eport |=
+        MG_EPHEMERAL_PORT_BASE;  // Random from MG_EPHEMERAL_PORT_BASE to 65536
```

Comment:
0 to 65535 and MG_EPHEMERAL_PORT_BASE to 65535

Reply:
Done